.github/workflows/unit-tests-rust.yaml

```diff
@@ -0,0 +1,52 @@
+name: unit-tests-rust
+run-name: Unit tests (rust) - ${{ github.run_id }} - @${{ github.actor }}
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    paths:
+      - '**.rs'
+      - 'Cargo.toml'
+      - 'Cargo.lock'
+
+defaults:
+  run:
+    working-directory: /workspace
+
+jobs:
+  test:
+    name: Unit tests (rust)
+    runs-on: ubuntu-latest
+    container: 
+      image: public.ecr.aws/gravitational/teleport-buildbox:teleport11
+      volumes:
+        - ${{ github.workspace }}:/workspace
+      options: --cap-add=SYS_ADMIN --privileged
+    steps:
+      - name: Install git 2.18+
```

Comment:
duplication

Reply:
Could you please explain?

Reply:
I think @gozer refers to the fact that this step is duplicated in all action .yaml files and wondering whether it can be encapsulated into "prepare-workspace" step.

Reply:
Great, v12 images have been updated. They are now used in master as primary. This step is not required anymore with :teleport12 and upper image versions. Removing everywhere along with upgrade to v12.